py/makeqstrdata.py

```diff
@@ -250,6 +250,73 @@
     "zip",
 ]
 
+# These qstrs have to be sorted early (preferably right after static_qstr_list) because they are required to fit in 8-bit values
```

Comment:
This is new right, so could you mark it with `CIRCUITPY-CHANGE`?

Reply:
happily, micropython took a version of this idea so I should go and cherry-pick theirs.